Dockerfile

```diff
@@ -70,6 +70,10 @@ RUN mkdir /miktex \
     && miktex --admin fndb refresh
 RUN chown -R $USERNAME:$USERNAME /miktex
 
+COPY --chown=$USERNAME:$USERNAME entrypoint.sh /home/$USERNAME
```

Comment:
```suggestion
COPY --chmod=1744 --chown=$USERNAME:$USERNAME entrypoint.sh /home/$USERNAME
```
chmod exsists in numerical form and works without buildkit special settings now.